CHANGELOG.md

```diff
@@ -3,6 +3,91 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
+## [0.8.0] - 2021-12-09
+### Compatibility Changes:
+- Platform support added:
+  - clang 12 #1385
+  - CUDA 11.4 #1380
+  - GCC 11 #1383
+  - HIP-clang #1338
+  - HIP-nvcc #1337
+  - Xcode 12.5.1 #1385
+  - Xcode 13 #1421
+- Platform support removed:
+  - clang < 5.0 #1385
+  - CUDA < 9.2 #1385
+  - GCC < 7.0 #1385
+  - `hipcc` #1338
+  - MSVC < 2019 #1385
+  - Ubuntu 16.04 #1352
+  - Xcode 11.x < 11.3.1 #1385
+  - Xcode 12.x < 12.4 #1385
+ 
+### Bug Fixes:
+- Added missing `#include <limits>` in a few places which would lead to compilation errors for CPU back-ends #1327
+- Added missing `std::` to fixed-width integers where necessary #1327
+- fixed behavior of `assert` and `printf` for OpenMP offloading targets #1351
+- `ALPAKA_STATIC_ACC_MEM_CONSTANT` now works correctly for clang-CUDA as well as HIP #1386
+- The OpenMP 5 and OpenACC back-ends now correctly pass the parameters as a `is_trivially_copyable` type #1387
+- Fixed `alpaka_compiler_option` checking for the wrong variable name #1392
+- Fixed a bug in the HIP back-end's peer-to-peer `memcpy` implementation #1400
+- The CMake function `alpaka_compiler_option` has been turned into a macro which solves parameter scope issues #1401
+- Fixed compilation error occuring with CUDA >= 11.3 #1404
+- the `-pthread` flag is now correctly passed to the (host) compiler and linker #1420
+- alpaka now correctly sets the CUDA host compiler #1423
+- alpaka's headers are now treated as CMake `SYSTEM` headers so that internal warnings no longer annoy users #1451
+- Projects using alpaka can now set `ALPAKA_CXX_STANDARD` without alpaka ignoring this #1463
+
+### New Features:
+- alpaka now supports the Philox random number generator #1319
+- alpaka's kernel language now supports memory fences (a.k.a thread fences) #1379
+- `alpaka::Vec` now supports structured bindings #1393
+- The OpenMP 5 and OpenACC back-ends now support statically mapped memory #1394
+- alpaka now has factory methods for creating memory views #1398
+- If OpenMP >= 5.1 is supported the back-end makes use of `atomic capture compare` #1411
+- alpaka now experimentally supports accessors instead of pointers to access memory #1433
+- added wrapper for CUDA's native vector types so that they may be handled like arrays #1435
+- added function for **exact** floating point comparisons #1440
```

Comment:
maybe mention that this is testing functionality.

Reply:
Why would it be testing functionality? It lives inside `alpaka::math` and is properly documented and unit tested.

Reply:
It is? I thought it was part of the unit test project. Ok, then leave it as it is!